DigitalLearningSolutions.Web/Helpers/UserTypes.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    public static class UserTypes

```

Comment:
I think Steve made an enum for admin vs delegate in https://github.com/TechnologyEnhancedLearning/DLSV2/pull/292,